### ISSUE TITLE
Debian bullseye: libpango1.0-0 dependency problem

### ISSUE THREAD

**Issue by JulienPalard** [dropbox/nautilus-dropbox#95]
Trying to install Dropbox on a Debian Bullseye:
```
$ sudo dpkg -i dropbox_2020.03.04_amd64.deb 
Selecting previously unselected package dropbox.
(Reading database ... 417780 files and directories currently installed.)
Preparing to unpack dropbox_2020.03.04_amd64.deb ...
Unpacking dropbox (2020.03.04) ...
dpkg: dependency problems prevent configuration of dropbox:
 dropbox depends on libpango1.0-0 (>= 1.36.3); however:
  Package libpango1.0-0 is not installed.
```

More info:
```
$ sudo apt install libpango1.0-0
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package libpango1.0-0 is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  libpangox-1.0-0

E: Package 'libpango1.0-0' has no installation candidate
```

**Comment by eddiesaliba** [comment]
I have the same problem.

**Comment by JulienPalard** [comment]
@eddiesaliba In the meantime I use dropbox-dist, from the bottom of this page: https://www.dropbox.com/install-linux
it does not needs libpango.

**Comment by eddiesaliba** [comment]
@JulienPalard thank you very much! 
After I wrote here I was looking for some solution and I found one that worked to me. It's described on this page: [https://www.reddit.com/r/debian/comments/g13vxj/dropbox_users_in_testingsid_libpango100_to/](url). It's very simple! Thanks again!

**Comment by jewelpit** [comment]
Fixed in #142.
